### DISCUSSION TITLE
hide orange dot in macOS Monterey

### DISCUSSION THREAD

**Discussion by es183923** [FelixKratz/SketchyBar#140] [Q&A] [answered]
macOS Monterey has this feature that shows an orange dot when your mic is being used by an app. Is it possible for sketchybar to be shown *over* this orange dot? It's very distracting to have a huge orange dot on the top right of my screen

![Screen Shot 2021-12-27 at 4 30 35 PM](https://user-images.githubusercontent.com/72744903/147508510-cf585bd3-1566-4839-b4b1-e48d57f3f00e.png)

**Comment by FelixKratz** [comment, marked as answer]
I have not tested this one: https://github.com/cormiertyshawn895/RecordingIndicatorUtility but it seems to do what you are asking for.
